toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1829,21 +1829,20 @@ func cleanupRpmDatabase(rootPrefix string) (err error) {
 	return
 }
 
-func RunPreInstallScripts(config configuration.SystemConfig) (err error) {
+func RunPreInstallScripts(config configuration.Config) (err error) {
 	const squashErrors = false
 
-	for _, script := range config.PreInstallScripts {
-		ReportActionf("Running pre-install script: %s", path.Base(script.Path))
-		logger.Log.Infof("Running pre-install script: %s", script.Path)
+	for _, sysConfig := range config.SystemConfigs {
```

Comment:
I think we would only want to run the pre-install scripts from the selected configuration. (not for `_,sysconfg := range config.Systemconfgs {`. We have a default system config entry `DefaultSystemConfig *SystemConfig` which should be populated if we don't know what we want to run. For unattended installs I believe we install the default config.

Reply:
Fixed